### PR TITLE
[A&I] async prod created topic

### PR DESCRIPTION
**Pull request by denisnikulin**

### CODE REVIEW

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: prod-aws
Module: pubsub/kafka-shared-msk-account-identity
Path: prod-aws/kafka-shared-msk/account-identity
Commit ID: 6209d82a36d2d218fceab08e218cae6c4d193002
Requested At: 2024-11-13T13:22:37Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan prod-aws/kafka-shared-msk/account-identity` as comment.

**Review by Jarota — APPROVED** [review]

File: prod-aws/kafka-shared-msk/account-identity/account.tf [hunk 1]
```diff
@@ -195,6 +195,12 @@ module "account_identity_account_api_v2" {
   cert_common_name = "account-platform/account_api_v2"
 }
 
+module "account_identity_account_api_v2_customer_bill_writer" {
+  source           = "../../../modules/tls-app"
+  produce_topics   = [kafka_topic.account_identity_accunt_bill_writes_public.name]
```
Comment (abramin) [inline prod-aws/kafka-shared-msk/account-identity/account.tf:200]: account_identity_account_bill_writes_public

**Comment by uw-infra** [comment]
Terraform run output for
```
Cluster: prod-aws
Module: pubsub/kafka-shared-msk-account-identity
Path: prod-aws/kafka-shared-msk/account-identity
Commit ID: 6fdd9fbf035088e7cac711f76a3fce28d362bca6
```
<details><summary><b>✅ Run Status: Ok, Run Summary: Apply complete! Resources: 1 added, 0 changed, 0 destroyed</b></summary>

```terraform

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.account_identity_account_api_v2_customer_bill_writer.kafka_acl.producer_acl["account-identity.account.bill.writes.public"] will be created
  + resource "kafka_acl" "producer_acl" {
      + acl_host                     = "*"
      + acl_operation                = "Write"
      + acl_permission_type          = "Allow"
      + acl_principal                = "User:CN=account-platform/account_api_v2_customer_bill_writer"
      + id                           = (known after apply)
      + resource_name                = "account-identity.account.bill.writes.public"
      + resource_pattern_type_filter = "Literal"
      + resource_type                = "Topic"
    }

Plan: 1 to add, 0 to change, 0 to destroy.
module.account_identity_account_api_v2_customer_bill_writer.kafka_acl.producer_acl["account-identity.account.bill.writes.public"]: Creating...
module.account_identity_account_api_v2_customer_bill_writer.kafka_acl.producer_acl["account-identity.account.bill.writes.public"]: Creation complete after 0s [id=User:CN=account-platform/account_api_v2_customer_bill_writer|*|Write|Allow|Topic|account-identity.account.bill.writes.public|Literal]

Warning: Argument is deprecated

  with provider["registry.terraform.io/mongey/kafka"],
  on __env.tf line 12, in provider "kafka":
  12: provider "kafka" {

This parameter is now deprecated and will be removed in a later release,
please use `client_cert` instead.

(and one more similar warning elsewhere)

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

```
</details>

> To manually trigger plan again please post `@terraform-applier plan prod-aws/kafka-shared-msk/account-identity` as comment.

File: prod-aws/kafka-shared-msk/account-identity/account.tf (hunk 1 above)
Comment (denisnikulin) [reply prod-aws/kafka-shared-msk/account-identity/account.tf:200]: https://github.com/utilitywarehouse/kafka-cluster-config/pull/697

**Pull request merged**